src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json

```diff
@@ -6,21 +6,16 @@
     ],
     "xmlDoc": true
   },
-  "version": "1.0.0-preview2-update1",
+  "version": "1.0.0-preview4-final",
   "description": "Contains utilities used by ASP.NET Core Code Generation packages.",
   "dependencies": {
     "Microsoft.CodeAnalysis.CSharp.Workspaces": "1.3.0",
     "Microsoft.DotNet.ProjectModel": "1.0.0-rc3-003121",
-    "Microsoft.Extensions.PlatformAbstractions": "1.0.0"
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0",
+    "NETStandard.Library": "1.6.0"
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Threading.Tasks": {
```

Comment:
Keeping this here and not adding NETStandard.Library 1.6.0 was causing build failure on my local about a missing System.* assembly reference for net451